src/apps/content-editor/src/app/views/ItemEdit/components/Header/ItemVersioning/ScheduleFlyout/ScheduleFlyout.js

```diff
@@ -200,14 +200,12 @@ export default class ScheduleFlyout extends Component {
                   />
                 </div>
                 <div className={styles.Row}>
-                  <FieldTypeDate
-                    type="date"
+                  <FieldTypeDateTime
                     name="publish"
                     label="Publish date and time"
-                    future={true}
```

Comment:
Does the MUI component allow selection of future values? We should include a test which confirms this, along side the existing past value test.

Reply:
It always allows the selection of future values. future={true} was a prop for the old component that avoided the selection of a past value

But yea I'll write some tests for this

Reply:
So the experience change introduced with this component swap is that past values are now selecteable? Do we care about this change? Can we prevent the new MUI component from allowing past value selections?

**Side note**: in regards to testing, what timezone is the browser test runner in github actions set to? What does that mean for our tests?

Reply:
No the experience is the same past values are still not selectable, this is set by a different prop 'disablePast'